pulp_2to3_migration/app/models/repository.py

```diff
@@ -178,4 +182,6 @@ class Meta:
         unique_together = ('pulp2_object_id',)
         indexes = [
             models.Index(fields=['pulp2_type_id']),
+            models.Index(fields=['pulp2_last_updated']),
+            models.Index(fields=['pulp2_repo_id']),
```

Comment:
Usage:
https://github.com/pulp/pulp-2to3-migration/blob/master/pulp_2to3_migration/app/migration.py#L212

Reply:
I found very similar for the importer, should I add the index to the importer as well?  
https://github.com/pulp/pulp-2to3-migration/blob/master/pulp_2to3_migration/app/migration.py#L165

Reply:
It's up to you - my personal view is that certain indexes for things like Repositories and Importers aren't *that* important comparatively since we are dealing with a much smaller number of them (vs. content, artifacts, LCEs)

But it proabably doesn't hurt.

Reply:
@goosemania If there are any queries that especially need to be fast, and they use multiple fields at once, we might want to add multi-column indexes (as opposed to indexes for multiple columns).

But it's easier for Postgresql to combine multiple indexes than it is to use a multi-column index for a single column search, so this is fine for now I think.  It's still better than having none at all as far as I can tell even if it's not optimally efficient.

https://stackoverflow.com/questions/39297221/multiple-indexes-vs-single-index-on-multiple-columns-in-postgresql
https://www.postgresql.org/docs/9.2/indexes-bitmap-scans.html
https://docs.djangoproject.com/en/3.1/ref/models/indexes/#condition